### PR TITLE
move _name mod to data-name attr

### LINKED ISSUE #173: move form-field_name_myName to data-bem props
Есть какой то смысл в физическом классе на элементе кроме как "удобно к нему обратиться" ? 

Выглядит не красиво. Более красиво брать имя из `this.props.name` 

Что думаете?

### PR DESCRIPTION
**Pull request by kompolom**
Close #173

### CODE REVIEW

File: common.blocks/form-field/form-field.bemhtml [hunk 1]
```diff
@@ -9,6 +9,7 @@ block('form-field')(
         _form_field.mods.name = _form_field.name;
```
Comment (voischev) [inline common.blocks/form-field/form-field.bemhtml]: Это зачем осталось?

**Comment by awinogradov** [comment]
👍

**Comment by awinogradov** [comment]
@zxqfox r?

File: common.blocks/form-field/form-field.browser.js [hunk 2]
```diff
@@ -15,7 +15,6 @@ provide(BEMDOM.decl(this.name, /** @lends form-field.prototype */{
                 this.__base.apply(this, arguments);
 
                 !this.hasMod('type') && console.warn('Type modifier required for form-field', this);
-                !this.hasMod('name') && console.warn('Name required for form-field', this);
```
Comment (zxqfox) [inline common.blocks/form-field/form-field.browser.js]: Is this not required? Not sure.

Comment (awinogradov) [inline common.blocks/form-field/form-field.browser.js]: +1

**Comment by zxqfox** [comment]
I guess we should revert drop `warn` for required name. All the rest looks good to me

**Comment by awinogradov** [comment]
@kompolom sorry! Can you return line with `warn` and squash your commits?

**Comment by kompolom** [comment]
Ok, just a moment..

**Comment by kompolom** [comment]
@awinogradov, @zxqfox Ready!

**Comment by awinogradov** [comment]
🔥

**Comment by adinvadim** [comment]
Why data-name instead `data-bem= "{name : }"`?

**Comment by awinogradov** [comment]
Faster search

**Pull request merged**